packages/devextreme-react/src/core/__tests__/template.test.tsx

```diff
@@ -14,7 +14,14 @@ import {
 
 jest.useFakeTimers();
 
-const waitForceUpdateFromTemplateRenderer = () => act(() => new Promise((ok: any) => requestAnimationFrame(ok)));
+const waitForceUpdateFromTemplateRenderer = () => {
+  act(() => {
+    new Promise((ok: any) => {
+      requestAnimationFrame(ok);
+    });
+  });
+}
```

Comment:
Some weird thing. The promise stopped being resolved at all when this was written in one line, but everything started working as expected when I simply added line breaks and curly braces. Looks like some transpilation artifact.